build/VStudio/testing/Directory.Build.props

```diff
@@ -0,0 +1,11 @@
+<Project>
```

Comment:
Is this file necessary? You can use file `build.common.props` to set properties. But in this case I would prefer to keep these properties in the `.csproj` file to minimize changes in this PR.